lib/Runtime/Library/Chakra.Runtime.Library.vcxproj

```diff
@@ -172,13 +172,15 @@
     <ClCompile Include="$(MSBuildThisFileDirectory)JSONStringBuilder.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)JSONStringifier.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)LazyJSONString.cpp" />
+    <ClCompile Include="PropertyRecordUsageCache.cpp" />
```

Comment:
nit: for razzle build to work, we need to prepend with $(MSBuildThisFileDirectory)

Reply:
Great catch! I just let VS do whatever it wanted to here, but it didn't match the correct style.

---
In reply to: [163133827](https://github.com/Microsoft/ChakraCore/pull/4569#discussion_r163133827) [](ancestors = 163133827)